sanic_jwt/configuration.py

```diff
@@ -66,10 +71,18 @@ def __iter__(self):
     def __repr__(self):
         return str(dict(iter(self)))  # noqa
 
+    def _merge_aliases(self, config):
```

Comment:
We can make this a `@staticmethod`

Reply:
_In theory_, yes. Practically? Not that much ... But since it's opensource, I'll stick to theory :sweat_smile:

Reply:
I mean, the effectiveness of that change ... If this method were inside `__init__`, the cost would be the same (just to clarify my comment above :laughing:) I think I'm not completely awake (yet) ... Now I'm in need of some ore coffee I guess :coffee: